samples/extensions/open_cl_common/README.md

```diff
@@ -0,0 +1,22 @@
+<!--
+- Copyright (c) 2023, Arm Limited and Contributors
+-
+- SPDX-License-Identifier: Apache-2.0
+-
+- Licensed under the Apache License, Version 2.0 the "License";
+- you may not use this file except in compliance with the License.
+- You may obtain a copy of the License at
+-
+-     http://www.apache.org/licenses/LICENSE-2.0
+-
+- Unless required by applicable law or agreed to in writing, software
+- distributed under the License is distributed on an "AS IS" BASIS,
+- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+- See the License for the specific language governing permissions and
+- limitations under the License.
+-
+-->
+
+# OpenCL common source files
+
+This folder does not contain a sample, but only OpenCL related units shared by the OpenCL interoperability samples in this repository.
```

Comment:
This would be good to move to components in then future

Reply:
Yeah, definitely. That would make a lot of sense. It does feel out of place in that folder, but it was the only place that made at least some sense.

Reply:
Happy with the current implementation so will resolve this comment